http/vibe/http/server.d

```diff
@@ -2427,11 +2427,37 @@ shared static this()
 	}
 }
 
-private string formatRFC822DateAlloc(IAllocator alloc, SysTime time)
+private struct CacheTime
+{
+	string cachedDate;
+	SysTime nextUpdate;
+
+	this(SysTime nextUpdate) @trusted
+	{
+		this.nextUpdate = nextUpdate;
+	}
+
+	void update(SysTime time) @trusted
+	{
+		this.nextUpdate = time + 1.seconds;
+		this.nextUpdate.fracSecs = nsecs(0);
+	}
```

Comment:
Why are those `@trusted` ? Additionally, this could be `@nogc pure nothrow` right ?